compiler/control/OMROptions.cpp

```diff
@@ -2604,6 +2605,8 @@ OMR::Options::jitPreProcess()
           ((TR::Compiler->target.cpu.isX86() && TR::Compiler->target.isLinux()) && TR::Compiler->target.numberOfProcessors() >= 4))
           self()->setOption(TR_TurnOffSelectiveNoOptServerIfNoStartupHint);
 
+      if(TR::Compiler->target.cpu.isX86() && TR::Compiler->target.is32Bit())
```

Comment:
can we have an envvar or something to override this for testing?

Reply:
I made this change in `jitPreProcess`, my understanding was that option parsing happens after this, so if someone passes `enableJProfilingInProfilingCompilations` it should reset the `TR_DisableJProfilingInProfilingCompilations` to enable jProfiling

Reply:
Oh I see - that I didn't know, but the options parsing is pretty complicated. Have you tested that works? If so then I will approve the change.

Reply:
Yeah, I tested this yesterday to make sure I have feature to allow us testing on platforms on which this is disabled. 
Looking at the code, 
https://github.com/eclipse/omr/blob/e54244dc64314b08585ddf6296ace2d490ad4544/compiler/control/OMROptions.cpp#L2360
In the function where we process `Xjit` options, we first call `jitPreProcess` first and then process the command line options. So any default option set would be overridden.